Makefile

```diff
@@ -5,6 +5,7 @@ PACKAGES := go list ./... | grep -v /vendor | grep -v /out
 
 
 product: clean
+	env GOOS=darwin GOARCH=amd64 go build -gcflags='-N -l' -o ${OUTPUT_DIR}/kubeturbo.osx ./cmd/kubeturbo
```

Comment:
`darwin` is not needed for product.

Reply:
For product no, this is for debugging. The final docker image won't contain it, and there's little danger in generating both.

Reply:
I think this is used locally and prefer not to commit.